src/main/java/com/minecolonies/coremod/blocks/BlockWaypoint.java

```diff
@@ -59,28 +59,29 @@ private void initBlock()
     }
 
     /**
-     * Used to determine ambient occlusion and culling when rebuilding chunks
-     * for render.
-     * @deprecated
-     *
-     * @return true
+     * @deprecated (Remove this as soon as minecraft offers anything better).
      */
-    //todo: remove once we no longer need to support this
     @SuppressWarnings(DEPRECATION)
     @Override
     @Deprecated
-    public boolean isOpaqueCube(final IBlockState state)
+    public boolean isFullBlock(final IBlockState state)
     {
         return false;
     }
 
     /**
-     * @deprecated (Remove this as soon as minecraft offers anything better).
+     * Used to determine ambient occlusion and culling when rebuilding chunks
+     * for render.
+     *
+     * @return true
+     *
+     * @deprecated
      */
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)